### PR TITLE
Redirect Done and Talk to Subject Post

### LINKED ISSUE #236: Done and Talk Should Post to Notes
Instead of linking to the talk board, done and talk should initialize a post for a particular subject. See PFE for example.

### PR DESCRIPTION
**Pull request by wgranger**
Instead of directing to the talk board, Done & Talk now redirects to a specific subject on the board. 

Closes #236

### CODE REVIEW

File: src/components/SubmitClassificationForm.jsx [hunk 1]
```diff
@@ -8,8 +8,7 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { config } from '../config';
 import {
-  createClassification, submitClassification,
-  setSubjectCompletionAnswers,
+  submitClassification, setSubjectCompletionAnswers,
```
Comment (wgranger) [inline src/components/SubmitClassificationForm.jsx]: `submitClassification` was not being used

Comment (shaunanoordin) [reply src/components/SubmitClassificationForm.jsx]: Wait, you mean `createClassification` was not being used?

Comment (wgranger) [reply src/components/SubmitClassificationForm.jsx]: Oh, my bad. Yes, `createClassification` is not being used by this component.

File: src/components/SubmitClassificationForm.jsx [hunk 2]
```diff
@@ -70,9 +69,13 @@ class SubmitClassificationForm extends React.Component {
       this.context.googleLogger.logEvent({ type: 'complete-classification-and-talk' });
     }
 
-    this.props.dispatch(submitClassification())
+    if (this.props.currentSubject) {
+      const id = this.props.currentSubject.id;
```
Comment (shaunanoordin) [inline src/components/SubmitClassificationForm.jsx]: Minor safety catch needed here - it's possible for `this.props.currentSubject` to be `null`

**Comment by wgranger** [comment]
Done. Created a general base url in the event of a null subject.

**Comment by shaunanoordin** [comment]
Noice! 👍  Let's get this merged.

**Pull request merged**